recommend/policyTemplates.go

```diff
@@ -0,0 +1,247 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of KubeArmor
+
+package recommend
+
+import (
+	"archive/zip"
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/cavaliergopher/grab/v3"
+	"github.com/google/go-github/github"
+	"github.com/kubearmor/kubearmor-client/selfupdate"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	org  = "kubearmor"
+	repo = "policy-templates"
+	url  = "https://github.com/kubearmor/policy-templates/archive/refs/tags/"
+)
+
+var currentVersion string
+
+// userHome function returns users home directory
+func userHome() string {
+	if runtime.GOOS == "windows" {
+		home := os.Getenv("HOMEDRIVE") + os.Getenv("HOMEPATH")
+		if home == "" {
+			home = os.Getenv("USERPROFILE")
+		}
+		return home
+	}
+	return os.Getenv("HOME")
+}
+
+// isTemplates function returns error if policy-template folder not found
+func isTemplates() error {
+	path := fmt.Sprintf("%s/.cache/karmor/", userHome())
```

Comment:
Keep the `.cache/karmor/` as a constant. Its better to write a function like `getCachePath()` and use it all the places.